### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mat-hek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ mix test
 
 # Copyright and License
 
-Copyright 2019, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=elixir-fdk-aac-docker)
+Copyright 2019, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=docker-ubuntu18-membrane)
 
-[![Software Mansion](https://membraneframework.github.io/static/logo/swm_logo_readme.png)](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=elixir-fdk-aac-docker)
+[![Software Mansion](https://membraneframework.github.io/static/logo/swm_logo_readme.png)](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=docker-ubuntu18-membrane
```
Comment (bblaszkow06) [inline README.md]: ubuntu18 is ambiguous. I would suggest using its codename (`bionic`)

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
+ENV BASH_ENV=~/.init
+
 SHELL ["/bin/bash", "-c"]
 
 RUN \
     #common
-    touch ~/.init && chmod u+x ~/.init && \
-    echo -e '\n. ~/.init' >> ~/.bashrc && \
+    echo -e '[[ -z $MEMBRANE_DOCKER_INIT ]] || return\nexport MEMBRANE_DOCKER_INIT=true' > ~/.init && \
```
Comment (bblaszkow06) [inline Dockerfile]: IMO unsetting `$BASH_ENV` would be easier to comprehend

File: Dockerfile [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+    chmod u+x ~/.init && . ~/.init && \
```
Comment (bblaszkow06) [inline Dockerfile]: This `chmod` is not needed

File: README.md (hunk 1 above)
Comment (mat-hek) [reply README.md]: I may use `ubuntu1804`

Comment (bblaszkow06) [reply README.md]: That seems off - ubuntu versions, especially in such use cases are referred by their codename, like in build environments in Travis https://docs.travis-ci.com/user/reference/xenial/

Comment (mat-hek) [reply README.md]: But I won't remember what is bionic in a few months

Comment (mat-hek) [reply README.md]: And most DockerHub repositories follow the numeric convention

Comment (bblaszkow06) [reply README.md]: Most? How have you measured that? From what I've seen they either support both (https://hub.docker.com/_/buildpack-deps) or only codenames (especially for Debian https://hub.docker.com/_/node)

Comment (bblaszkow06) [reply README.md]: And if you insist on using numbers, make it "ubuntu-18.04-membrane"

Comment (mat-hek) [reply README.md]: Yes I have :D
https://hub.docker.com/search?q=ubuntu18&type=image
https://hub.docker.com/search?q=bionic&type=image
18.04 can be

Comment (bblaszkow06) [reply README.md]: Have you noticed the popularity of images under both links? :stuck_out_tongue:

Comment (mat-hek) [reply README.md]: Have you noticed that the most popular is and image of PHP? :D

Comment (mat-hek) [reply README.md]: ok, I can change to this bionic... but that bionic-membrane or ubuntu-bionic-membrane?

**Pull request merged**